scipy/_lib/tests/test_array_api.py

```diff
@@ -68,6 +69,10 @@ def test_copy(self, xp):
             x[1] = 11
             x[2] = 12
 
+            if is_dask(xp):
+                x.compute()
+                y.compute()
```

Comment:
Think these should be assigned to variables (possibly the same variables they already occupy like `x = x.compute()`)

Also the `id` check at the end makes me wonder if this should use `.persist()` instead of `.compute()` (the previous point would still apply)